apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,355 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  AvatarBaseShape,
+  AvatarToken,
+  AvatarTokenSize,
+  BadgeNetwork,
+  ButtonSize,
+  ButtonIcon,
+  ButtonIconSize,
+  IconName,
+  Text,
+  TextVariant,
+  TextColor,
+  AvatarAccount,
+  AvatarAccountVariant,
+  Icon,
+  ButtonBase,
+  Box,
+  BoxBackgroundColor,
+  BoxBorderColor,
+} from '@metamask/design-system-react';
+
+const meta: Meta = {
+  title: 'Examples/Wallet Home',
+  component: () => null,
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+const WalletHome: React.FC = () => {
+  return (
+    <div className="bg-background-default min-h-screen">
+      {/* Header */}
+      <Box
+        backgroundColor={BoxBackgroundColor.BackgroundDefault}
+        padding={4}
+        borderColor={BoxBorderColor.BorderMuted}
+      >
+        <div className="flex items-center justify-between">
+          <div className="flex items-center space-x-2">
+            <AvatarAccount
+              shape={AvatarBaseShape.Square}
+              variant={AvatarAccountVariant.Maskicon}
+              address=""
+            />
+            <Text variant={TextVariant.HeadingMd} color={TextColor.TextDefault}>
+              DeFi Account
+            </Text>
```

Comment:
Is the idea to eventually move this tag to a more web centric `h3` tag or whatever, along with tailwind styles?